backend/models/services.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Defines the types of services available to homeowners and contractors.
+ */
+
+
+// const SERVICES = new Set([
+//     SERVICE_ROOFING,
+//     SERVICE_REMODELING,
+//     SERVICE_PLUMBING,
+//     SERVICE_PEST_CONTROL,
+//     SERVICE_LANDSCAPING,
+//     SERVICE_ELECTRICAL
+// ]);
+
+/**
+ * Checks whether the given array of services contains only valid service types.
+ * @param {Array[String]} services 
+ * @returns true if the array of service types contains only valid service types, false otherwise
+ */
+// function validateServices(services) {
+//     for (const service of services) {
+//         if (!SERVICES.has(service.toLowerCase())) {
+//             return false;
+//         }
+//     }
+//     return true;
+// }
+
+// module.exports = {
+//     SERVICES: SERVICES,
+//     SERVICE_ROOFING: SERVICE_ROOFING,
+//     SERVICE_REMODELING: SERVICE_REMODELING,
+//     SERVICE_PLUMBING: SERVICE_PLUMBING,
+//     SERVICE_PEST_CONTROL: SERVICE_PEST_CONTROL,
+//     SERVICE_LANDSCAPING: SERVICE_LANDSCAPING,
+//     SERVICE_ELECTRICAL: SERVICE_ELECTRICAL,
+//     validateServices: validateServices
+// }
+
+// Permitted service types
+const services = Object.freeze({
+  PLUMBING: "Plumbing",
+  REMODELING: "Remodeling",
```

Comment:
Do we want the service values to be case sensitive? If so, the frontend needs to be aware of this so that it can send over correctly cased values (e.g., it shouldn't send something like "plumbing").

Reply:
It totally depends on how the front end wants to send the request. If we define the input id as "Plumbing" then this code is fine. Or do you want it to be "plumbing"?

Reply:
Ok let me quickly update it to all lower case as that's more convenient.